apps/nextjs-website/src/lib/fetch/types/PageSection.ts

```diff
@@ -170,6 +170,13 @@ const FormSectionCodec = t.strict({
   submitHandler: t.Function,
 });
 
+const PreFooterSectionCodec = t.strict({
+  __component: t.literal('sections.pre-footer'),
+  title: t.string,
+  theme: t.union([t.literal('light'), t.literal('dark')]),
+  storeButtons: t.union([StoreButtonsCodec, t.null]),
+});
```

Comment:
From this type it looks like we are never going to pass a `decoration` to `PreFooterRC`.

We should either add `decoration` here or remove it entirely from `PreFooterRC` if it's not going to be used.